pantheon-bundle/frontend/src/app/moduleDisplay.tsx

```diff
@@ -306,6 +306,10 @@ class ModuleDisplay extends Component<any, IModuleDisplayState> {
 
     private onPublishEvent = () => {
         this.getVersionUUID(this.props.location.pathname)
+        // wait for 1 second so that JCR data gets updated
```

Comment:
I don't believe this fix the underlining problem. The reason on Module display, "View on Customer Portal" link doesn't show up when publishing is because, you need add this method call in public componentDidMount() method:
  this.getPortalUrl()

Reply:
This was based on the existing code and not the new code that has just been merged. Will make changes accordingly. Also, there is a time gap between 200 OK received and the actual JCR updates happening in the backend. I will setup a meeting on Monday to discuss this PR.